src/boost_histogram/cpp/algorithm.py

```diff
@@ -56,13 +56,6 @@ def empty(histogram, flow=False):
     return histogram._empty(flow)
 
 
-def reduce(histogram, *args):
```

Comment:
Just curious, isn't this needed to match the C++ interface?

Reply:
It was defined twice in this file.

Reply:
same function, just different doc string.

Reply:
Good catch!